### ISSUE TITLE
The bot run normally when i setup, but an error happened when i excute "buy" order.

### ISSUE THREAD

**Issue by Trinhhoanglong1610** [Nafidinara/bot-pancakeswap#86]
Hello, please check this out, it pop up this error "Could not detect network" when i testing to buy a token. 

![12345](https://user-images.githubusercontent.com/88970314/131204828-c58ddabf-d4b3-4cef-85e7-1959811d59d3.PNG)

**Comment by Nafidinara** [comment]
please contact me on discord : countdown#4008

Pada tanggal Sab, 28 Agu 2021 pukul 10.27 Trinhhoanglong1610 <
***@***.***> menulis:

**Comment by Trinhhoanglong1610** [comment]
I sent you a request on Discord, thank you for your helping.

**Comment by Nafidinara** [comment]
or please contact on (A algarox#0785)

Pada tanggal Sab, 28 Agu 2021 pukul 16.12 Trinhhoanglong1610 <
***@***.***> menulis:

**Comment by Trinhhoanglong1610** [comment]
Yes, i did send request for both of the account, please check, thank you 👯‍♂️
